docusaurus.config.js

```diff
@@ -104,7 +104,7 @@ const config = {
         ],
       },
       footer: {
-        style: 'dark',
```

Comment:
Please revert the changes on the footer. This is intended at https://github.com/apache/incubator-kvrocks-website/pull/69. We should avoid being up and down.

Reply:
Sorry, I didn't notice #69 before. The footer has been reverted now.